src/anim/controller/anim-blend-tree.js

```diff
@@ -66,6 +71,10 @@ class AnimBlendTree extends AnimNode {
         return paramsEqual;
     }
 
+    getNodeWeightedDuration(i) {
+        return this._children[i].animTrack.duration / this._children[i].speedMultiplier * this._children[i].weight;
```

Comment:
store this in a variable 
`this._children[i]`

Reply:
Sure